supported-os-installer.sh

```diff
@@ -0,0 +1,524 @@
+#!/usr/bin/env bash
+
+# This script is mostly based on ->
+# https://github.com/Zimbra/zinstaller/blob/main/zinstaller script
+
+# Warnings :	This script is still a WORK IN PROGRESS. Not fully tested.
+#		Script to install Zimbra Classic - ONLY SINGLE NODE on a provided remote server
+############################################################################################
+
+function display_options {
+cat << HELP_EOF
+	"$0" -i <install-config> -l <license> -t <build-artifact> -a <admin-pass> --optional-pkg <pkg1> --optional-pkg <pkg2>
+	options:
+	-h, --help			Help
+	-p, --admin-pass		Admin account password to set. (Default: Random)
+	-l, --license			License key or License file to be activated on the zimbra server. (Required) (No default)
```

Comment:
I think barry's script supports FOSS install also, so we should also support it

Reply:
FOSS was not supported in original script, which was mostly written by Gautam, not me. 

A trial license was automatically obtained and installed. This needs to be re-implemented, but the current license server does not support it.

Reply:
ok then we will go ahead with current implementation and then in future can revisit and add this support as well